samples/sensor/thermometer/sample.yaml

```diff
@@ -10,6 +10,7 @@ tests:
       - robokit1              # ntc_thermistor
       - adi_eval_adin1110ebz  # adt7420
       - frdm_mcxn947/mcxn947/cpu0 # p3t1755
+      - frdm_mcxa156          # p3t1755
```

Comment:
Drop. There is no need to add to CI load as this will not be testing anything that isn't already tested elsewhere (i3c driver tests, sensor driver test, or even the line just above--even if it should likely also go away)
https://docs.zephyrproject.org/latest/samples/sample_definition_and_criteria.html#twister-should-be-able-to-build-every-sample